### PR TITLE
1360 Correct TeeInput tests

### PR DESCRIPTION
**Pull request by a-korzhov**
#1360
 - Correct tests of TeeInputs

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#1370](https://github.com/yegor256/cactoos/issues/1370) is assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/C63314D6Z/da3134bf-872e-4edb-b094-d9af4cdda9c4); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java [hunk 1]
```diff
@@ -52,13 +53,12 @@ public void copiesFromCharArrayWithCharsetToFile() throws IOException {
         final String input =
             "Hello, товарищ file #1 äÄ üÜ öÖ and ß";
         final File output = this.folder.newFile();
+        new LengthOf(
+            new TeeInput(input.toCharArray(), new OutputTo(output))
```
Comment (victornoel) [inline src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java:57]: @a-korzhov you mis-copied it, the charset parameter is missing

File: src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java [hunk 2]
```diff
@@ -85,12 +84,12 @@ public void copiesFromCharArrayToOutput() throws IOException {
         final String input =
             "Hello, товарищ output #1 äÄ üÜ öÖ and ß";
         final File output = this.folder.newFile();
+        new LengthOf(
+            new TeeInput(input.toCharArray(), new OutputTo(output), StandardCharsets.UTF_8.name())
```
Comment (victornoel) [inline src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java:88]: @a-korzhov you mis-copied it, you added a charset parameter while it wasn't here before

File: src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java [hunk 3]
```diff
@@ -69,13 +69,12 @@ public void copiesFromCharArrayWithCharsetByNameToFile()
         final String input =
             "Hello, товарищ file #2 äÄ üÜ öÖ and ß";
         final File output = this.folder.newFile();
+        new LengthOf(
+            new TeeInput(input.toCharArray(), new OutputTo(output), StandardCharsets.UTF_8.name())
```
Comment (victornoel) [inline src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java:73]: @a-korzhov youchanged the code, it was `output`, not `new OutputTo(output)`

File: src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java [hunk 4]
```diff
@@ -100,13 +99,12 @@ public void copiesFromCharArrayWithCharsetToOutput() throws IOException {
         final String input =
             "Hello, товарищ output #2 äÄ üÜ öÖ and ß";
         final File output = this.folder.newFile();
+        new LengthOf(
+            new TeeInput(input.toCharArray(), new OutputTo(output), StandardCharsets.UTF_8.name())
```
Comment (victornoel) [inline src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java:103]: @a-korzhov you added `.name()`, it wasn't there before

File: src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java [hunk 5]
```diff
@@ -133,12 +130,12 @@ public void copiesFromCharArrayToPath() throws IOException {
         final String input =
             "Hello, товарищ path #1 äÄ üÜ öÖ and ß";
         final File output = this.folder.newFile();
+        new LengthOf(
+            new TeeInput(input.toCharArray(), new OutputTo(output), StandardCharsets.UTF_8.name())
```
Comment (victornoel) [inline src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java:134]: @a-korzhov here you added charset that shouldn't be here

File: src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java [hunk 6]
```diff
@@ -148,13 +145,12 @@ public void copiesFromCharArrayWithCharsetToPath() throws IOException {
         final String input =
             "Hello, товарищ path #2 äÄ üÜ öÖ and ß";
         final File output = this.folder.newFile();
+        new LengthOf(
+            new TeeInput(input.toCharArray(), new OutputTo(output), StandardCharsets.UTF_8.name())
```
Comment (victornoel) [inline src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java:149]: @a-korzhov here too

File: src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java [hunk 7]
```diff
@@ -165,13 +161,12 @@ public void copiesFromCharArrayWithCharsetByNameToPath()
         final String input =
             "Hello, товарищ path #3 äÄ üÜ öÖ and ß";
         final File output = this.folder.newFile();
+        new LengthOf(
+            new TeeInput(input.toCharArray(), new OutputTo(output), StandardCharsets.UTF_8.name())
```
Comment (victornoel) [inline src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java:165]: @a-korzhov here too

File: src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java [hunk 8]
```diff
@@ -181,12 +176,12 @@ public void copiesFromCharArrayToFile() throws IOException {
         final File output = this.folder.newFile();
         final String input =
             "Hello, товарищ file äÄ üÜ öÖ and ß";
+        new LengthOf(
+            new TeeInput(input.toCharArray(), new OutputTo(output), StandardCharsets.UTF_8.name())
```
Comment (victornoel) [inline src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java:180]: @a-korzhov here too

File: src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java (hunk 7 above)
Comment (a-korzhov) [reply src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java:165]: > @a-korzhov here too

Guess I understood the logic, but then, why I dont have to put charset here, if the message is '... copied with UTF_8 charsets name'?

Comment (victornoel) [reply src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java:165]: @a-korzhov I meant "here too there is a problem". Just take care of not changing the test behaviour, only that it is correctly verifying the output content as per the `@todo` description

Comment (a-korzhov) [reply src/test/java/org/cactoos/io/TeeInputFromCharArrayTest.java]: @victornoel Done, check please.

**Comment by victornoel** [comment]
@a-korzhov great thx

@paulodamaso good to merge

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 39 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @victornoel[/z](https://www.0crat.com/u/victornoel), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@a-korzhov please make sure you start all your comments with the name of the user they are referred to, [see](https://github.com/yegor256/cactoos/pull/1370#discussion_r418978817)

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**